shared/team-building/search-result/user-result.tsx

```diff
@@ -15,15 +15,26 @@ const UserResult = React.memo((props: ResultProps) => {
       rowStyle={styles.rowContainer}
       rightButtons={
         !props.isPreExistingTeamMember && (
-          <ActionButton inTeam={props.inTeam} onAdd={props.onAdd} onRemove={props.onRemove} />
+          <ActionButton
+            inTeam={props.inTeam}
+            onAdd={(e: React.BaseSyntheticEvent) => {
+              e.stopPropagation()
+              props?.onAdd()
```

Comment:
I don't understand what this is protecting against.

Reply:
Onadd was always optional. Since we're doing 2 things we need to check